Language/Generics/Superbounded_types/class_A03_t02.dart

```diff
@@ -7,8 +7,8 @@
 ///   ...
 /// - `T` is an immediate subterm of a redirecting factory constructor signature
 ///
-/// @description Checks that it is a compile-time error if a type `T` is used in
-/// the redirecting factory constructor signature
+/// @description Checks that it is a compile-time error if a super-bounded type
+/// is an immediate subterm of a redirecting factory constructor signature
```

Comment:
This test needs the same fix as Language/Generics/Superbounded_types/class_A03_t01.dart.

Reply:
Fixed.